src/inference/src/cpp/ie_executable_network.cpp

```diff
@@ -247,9 +247,15 @@ Any CompiledModel::get_property(const std::string& name) const {
             }
         }
         try {
-            return {_impl->GetMetric(name), {_so}};
+            if (target_device.empty())
+                return {_impl->GetMetric(name), {_so}};
```

Comment:
If you call get_property(ov::device::properties("CPU", ov::num_streams)) from a CPU executable_network, what will happen? I believe exception will throw as CPU does not implement the new getConfig API.

Reply:
Yes.